docs/keyboard-shortcuts.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Keyboard Shortcuts
+---
+
+export const Shortcut = ({ children }) => (
+  <kbd
+    style={{
+      margin: '0.2rem 0rem',
+    }}
+  >
+    {children}
+  </kbd>
+);
+
+A list of keyboard shortcuts that can be used within the Aerie UI.
+
+### Plan View
```

Comment:
Missing command/ctrl + s for simulate in the plan view

Reply:
I added this one, but i think there was discussion about actually removing the shortcut. Since we have it probably good to document it.